src/js/wrappers/ListWrapper.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+const ListWrapper = ({ children }) => (
+  <div className="d-flex flex-column list-page-main">
+    {children}
+  </div>
+);
+
+export default ListWrapper;
+
+ListWrapper.propTypes = {
+  children: PropTypes.node.isRequired,
+};
```

Comment:
the purpose of all those wrappers is that we have the same `<div>` with the same classNames applied for all of the list pages, so if we wanted e.g. to change any flex settings or anything, we'd have to go through all of the list pages and change it manually.
Those wrappers could be used for all of the list pages we've recently implemented. If you like the idea, we could create a small tech debt ticket to replace the `<div>`s with those reusable wrappers.

Reply:
Sounds good, I think we should do it in a separate ticket ASAP.
This change is not complicated and should not take to long to implement, I would hate it if these components were used in one List page and not the rest.
I think they should also be able to receive an optional `className` prop to have the ability to add custom classes like
```jsx
const ListWrapper = ({ children, className }) => (
  <div className={`d-flex flex-column list-page-main ${className}`}>
    {children}
  </div>
);

export default ListWrapper;

ListWrapper.defaultProps = {
  className: "",
}

ListWrapper.propTypes = {
  children: PropTypes.node.isRequired,
  className: PropTypes.string,
};
```

Reply:
good idea